components/admin_console/admin_definition.jsx

```diff
@@ -3417,6 +3440,7 @@ const AdminDefinition = {
             title_default: 'AD/LDAP',
             isHidden: it.any(
                 it.licensedForFeature('LDAP'),
+                it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.AUTHENTICATION.LDAP)),
```

Comment:
I don't think we want a check on this resource. I believe everyone should be able to see it.